src/flavors/bellevue-bike-share/config.json

```diff
@@ -695,6 +695,25 @@
           }
         ],
         "fields": [
+          {
```

Comment:
@jalMogo -- FYI here is the config I put together for the "advance to arbitrary stage" feature. This is all temporary so it can be changed around if need be. Note that we're sort of hacking the toggle field to achieve the effect of a button that advances the form's stage. In the refactor I think an advance stage "field" should actually be a button, though the backend doesn't need to be concerned about that.